api/src/main/java/org/openmrs/module/openconceptlab/ImportServiceImpl.java

```diff
@@ -318,6 +318,9 @@ public Subscription getSubscription() {
 		String token = adminService.getGlobalProperty(OpenConceptLabConstants.GP_TOKEN);
 		subscription.setToken(token);
 
+		String validation = adminService.getGlobalProperty(OpenConceptLabConstants.GP_VALIDATION);
+		subscription.setValidation(Boolean.valueOf(validation));

```

Comment:
It shouldn't be boolean as we will want to introduce more validation options. Make it an enum with "NONE" and "FULL" for now.